### PR TITLE
added account name to WindowTitle

### PR DESCRIPTION
**Pull request by BreezeLo**
added account name to the WindowTitle in "UpdateLiveStats" task, easier to manage multiple bots and their outputs.

### CODE REVIEW

**Comment by BriceSD** [comment]
@VincentKO Looks like a feature request, not a pull request.

**Comment by BreezeLo** [comment]
files changed are above, pardon me if that was done wrong https://github.com/PokemonGoF/PokemonGo-Bot/pull/3876/files

**Comment by BriceSD** [comment]
@BreezeLo I thought the username wasn’t available and you were asking us to add it. Sorry I misundertood.

**Comment by BreezeLo** [comment]
I wrote "add" instead of added, sorry about that. Waiting for approval

File: configs/config.json.example [hunk 1]
```diff
@@ -31,9 +31,10 @@
       {
         "type": "UpdateLiveStats",
         "config": {
+
```
Comment (TheSavior) [inline configs/config.json.example]: please remove this blank line

Comment (BreezeLo) [inline configs/config.json.example]: removed (a709c6c)

**Comment by TheSavior** [comment]
:+1:

**Pull request merged**